admin/admin.go

```diff
@@ -16,17 +16,18 @@ import (
 )
 
 type Options struct {
-	DatabaseDriver     string
-	DatabaseDSN        string
-	ExternalURL        string
-	FrontendURL        string
-	ProvisionerSetJSON string
-	DefaultProvisioner string
-	VersionNumber      string
-	VersionCommit      string
-	MetricsProjectOrg  string
-	MetricsProjectName string
-	AutoscalerCron     string
+	DatabaseDriver            string
+	DatabaseDSN               string
+	ExternalURL               string
+	FrontendURL               string
+	ProvisionerSetJSON        string
+	DefaultProvisioner        string
+	VersionNumber             string
+	VersionCommit             string
+	MetricsProjectOrg         string
+	MetricsProjectName        string
+	AutoscalerCron            string
+	DatabaseEncryptionKeyring string
```

Comment:
nit: move up below the `DatabaseDSN` so we group database-related properties together